docs/wiki/Download-and-Install.md

```diff
@@ -0,0 +1,206 @@
+<a id="top"></a>
+
+_Last updated for Bison Wallet v1.0.0._
+
+
+**Contents**
+- [Download](#download)
+- [Bison Wallet Desktop](#bison-wallet-desktop)
+- [Bison Wallet CLI](#bison-wallet-cli)
+- [Building from Source](#building-from-source)
+- [Docker](#docker)
+
+
+# Download
+
+The latest version of Bison Wallet can be downloaded from https://dex.decred.org.
+
+> [!NOTE]
+> We recommend you also verify that your download hash matches the hash 
+in the DCRDEX releases manifest. For detailed instructions, read about 
+[Verifying Binaries](https://docs.decred.org/advanced/verifying-binaries/)
+ in the Decred Documentation.
+
+You will need to visit the [releases](https://github.com/decred/dcrdex/releases) page 
+to download the manifest and manifest signature:
+
+```
+bisonw-v1.0.0-manifest.txt
+bisonw-v1.0.0-manifest.txt.asc
+```
+
+# Bison Wallet Desktop
+
+``bisonw-desktop`` is the Desktop version of Bison Wallet. This version is a self-contained 
+application,  making it the preferred option for new users.
+
+
+<details>
+<summary><b>Windows</b></summary> 
+
+1. Download the Windows installer ``bisonw-desktop-windows-amd64-v1.0.0.msi``.
+
+2. Double click the installer and follow the instructions.
+
+3. The installer adds a shortcut to Bison Wallet on your Start Menu.
+
+</details>
+<details>
+<summary><b>macOS</b></summary> 
+
+1. Download the ``bisonw-desktop-darwin-amd64-v1.0.0.dmg`` file.
+
+2. Double click the ``bisonw-desktop-darwin-amd64-v1.0.0.dmg` file to mount the disk image.
+
+3. Drag the ``bisonw-desktop.app`` file into the link to your Applications folder within the 
+disk image.
+
+</details>
+<details>
+<summary><b>Linux (Debian / Ubuntu)</b></summary> 
+
+1. Download the ```bisonw-desktop-linux-amd64-v1.0.0.deb``` file.
+
+2. Open a terminal in the extracted folder and run the command 
+```dpkg -i ./bisonw-desktop-linux-amd64-v1.0.0.deb```.
```

Comment:
`sudo` dpkg -i ...

Not sure if we need to explain what sudo is?

Reply:
added the sudo, not sure if we need to add that level of detail